modules/validations/incompatible_source.js

```diff
@@ -8,7 +8,7 @@ export function validationIncompatibleSource() {
   const incompatibleRules = [
     {
       id: 'amap',
-      regex: /(amap|autonavi|mapabc|高德)/i
+      regex: /(^amap$|amap\.at|^amap\.com|autonavi|mapabc|高德)/i
```

Comment:
Can you please drop `amap.at` from this list? It doesn't have anything to do with the data from [AutoNavi](https://en.wikipedia.org/wiki/AutoNavi), which this rule is obviously about. The amap.at domain is owned by an Austrian government agency.

@mmd-osm if you'd like to see this URL added to the list of incompatible sources, please open a new PR (or issue) for it. :blush: At first glance, amap.at seems to be a former, now dysfunctional, URL of what is now available at http://www.austrianmap.at/amap/, which AFAICS does contain non-open data (e.g., `ÖK 50`).